docs/reference/index-modules/history-retention.asciidoc

```diff
@@ -65,6 +65,9 @@ there>>. {ccr-cap} will not function if soft deletes are disabled.
   configured at index creation and only on indices created on or after 6.5.0.
   The default value is `true`.
 
+  deprecated::[7.6, Indices with soft deletes disabled are deprecated and will be rejected.
```

Comment:
> and will be rejected

This sounds as if these indices will no longer be supported. Perhaps reword this to: `Creating indices with soft-deletes disabled is deprecated and will be removed in future Elasticsearch versions.`

Reply:
++. Adjusted in https://github.com/elastic/elasticsearch/pull/50502/commits/867050d34b0d56783e11f6435afa8d1809bc157c.